.circleci/config.yml

```diff
@@ -1278,6 +1275,7 @@ workflows:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - release-checks:
+          xcode_version: '14.3.0'
```

Comment:
This was failing due to visionos not being available in intel machines. Also carthage doesn't work with M1 machines, so we can't just upgrade to M1 and have visionOS and run carthage

We need to figure out what we do with carthage, because Intel machines are going away June 28